### PR TITLE
added 21.1 hw solution

### PR DESCRIPTION
**Pull request by Tarabarnik**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    private static final String DB_NAME = "items";
+
+    public ItemDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    @Override
+    public Item add(Item item) {
+        Statement statement = null;
+        String query = "INSERT INTO " + DB_NAME + " (name, price)" + " VALUES " + "('"
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: try to use String.format in all methods

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 2]
```diff
@@ -44,8 +46,8 @@ public Item add(Item item) {
     @Override
     public Item get(Long id) {
         Statement statement = null;
-        String query = "SELECT * FROM " + DB_NAME + " WHERE item_id=" + id + ";";
-
+        Locale.setDefault(Locale.US);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: do we need it here?

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 3]
```diff
@@ -94,7 +97,8 @@ public Item update(Item item) {
     @Override
     public void delete(Long id) {
         Statement statement = null;
-        String query = "DELETE FROM " + DB_NAME + " WHERE item_id=" + id + ";";
+        Locale.setDefault(Locale.US);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: the same:)

**Review by Sofasmile — APPROVED** [review]

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java (hunk 2 above)
Comment (Tarabarnik) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: We don't. Sorry, a bit sleepy today

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java (hunk 3 above)
Comment (Tarabarnik) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: same:)

**Pull request merged**